### DISCUSSION TITLE
Related to VSCODE and Quarto.

### DISCUSSION THREAD

**Discussion by DrDinhLuong** [quarto-dev/quarto-cli#9625] [Q&A] [answered]
### Description

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Applications/quarto/share/editor/tools/vs-code.

Pls help me

**Comment by cscheid** [comment, marked as answer]
Sorry, but we need more information than this to be able to help. What were you trying to do? Where does this error appear? Are there more messages? I see a mismatched `'` so it looks like there's some copy-pasting error.

**Reply by DrDinhLuong** [reply]
version Quarto: 1.3.290
Vscode: 1.89.0

When i open vscode and create new Quarto project then show to me new notification in this figure. 
<img width="384" alt="Screenshot 2024-05-09 at 13 26 39" src="https://github.com/quarto-dev/quarto-cli/assets/86107710/9ad339db-de5d-440a-984f-feb65aeed0f4">
I don't know should I do? pls help me

**Reply by DrDinhLuong** [reply]
This is full output

<img width="748" alt="Screenshot 2024-05-09 at 13 28 48" src="https://github.com/quarto-dev/quarto-cli/assets/86107710/40eb8b83-613a-4e21-bcf1-91d1cb69a44a">

**Reply by cscheid** [reply]
Try updating Quarto: https://quarto.org/docs/get-started/

**Reply by DrDinhLuong** [reply]
Thank you so much. I got it.
So, other problems come to me. Because when I click to "Run cell" it not show result. 
<img width="1091" alt="Screenshot 2024-05-09 at 16 08 57" src="https://github.com/quarto-dev/quarto-cli/assets/86107710/34621774-c737-4a79-a357-37684370045c">
Please help me.

**Reply by DrDinhLuong** [reply]
This is notification
<img width="374" alt="Screenshot 2024-05-09 at 16 45 37" src="https://github.com/quarto-dev/quarto-cli/assets/86107710/41456696-9eb6-4b8f-bc78-c3d3485b975b">

**Comment by DrDinhLuong** [comment]
Thank you so much. I fixed it. 
we can close

**Comment by DrDinhLuong** [comment]
Thank you so much
